### PR TITLE
Urn references

### PR DESCRIPTION
**Pull request by anthonyroach**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (tdstein) [inline .vscode/settings.json]: Can you add this file to the .gitignore?

**Review by tdstein — APPROVED** [review]
LGTM

**Review by dmmiller612 — APPROVED** [review]

File: src/main/scala/com/lifeomic/fhirlib/v3/datatypes/Reference.scala [hunk 2]
```diff
@@ -8,6 +8,11 @@ class Reference(val reference: Option[String],
       val uri = new java.net.URI(reference.get)
       if (uri.getFragment != null) {
         return Some(uri.getFragment)
+      } else if (uri.getScheme == "urn" && uri.getSchemeSpecificPart.split(":")(0) == "uuid") {
+        val parts = uri.getSchemeSpecificPart.split(":");
+        if (parts(0) == "uuid") {
```
Comment (mjtieman) [inline src/main/scala/com/lifeomic/fhirlib/v3/datatypes/Reference.scala:13]: Don't need this check, already done in the if statement

Comment (anthonyroach) [reply src/main/scala/com/lifeomic/fhirlib/v3/datatypes/Reference.scala:13]: oops, meant to remove that

**Review by mjtieman — APPROVED** [review]
LGTM

**Pull request merged**